README.md

```diff
@@ -28,15 +28,15 @@ To run the project , follow the instructions below .
    Note:
  * The OpenMRS Instance is pre-loaded with CIEL and a  sample form (HIV_form) to collect TX_PVLS specific data
 
- * The above script also loads the necesary TX_PVLS Measure and Library Resources into the Hapi FHir . see the Resources under the `resources` folder.
+ * The above script also loads the necesary TX_PVLS,TX_CURR Measures and Library Resources into the Hapi FHir . see the Resources under the `resources` folder.
 
  * The Openhim instance is pre-configured with the necesary meta-data
 
   > Wait for the **plir-streaming-pipeline** container to start running before adding any data into OpenMRS. The running Pipeline will then listen to any  data changes  added in to OpenMRS and route them to the FHIR server through OpenHIM.
 
    
 
-  4. Invoke the **collect-data** FHIR Operation using the GET request below to generate the relevant Dataset for TX_PVLS
+  4. Invoke the **collect-data** FHIR Operation using the GET request below to generate the relevant Dataset for TX_PVLS or TX_CURR
 
 
           GET: http://localhost:8090/fhir/Measure/TX-PVLS/$collect-data?periodStart=<date>&periodEnd=<date>
```

Comment:
can you also put the endpoint to invoke the collect-data  operation for generating a daset for `TX_CURR` ??